<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LampHistoryPageClass</class>
 <widget class="QWidget" name="LampHistoryPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="TableCtrl" name="tableView"/>
   </item>
   <item row="1" column="0">
    <widget class="BtnCtrl" name="firstBtn">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="BtnCtrl" name="prevBtn">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="BtnCtrl" name="nextBtn">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="BtnCtrl" name="lastBtn">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtnCtrl</class>
   <extends>QPushButton</extends>
   <header>btnctrl.h</header>
  </customwidget>
  <customwidget>
   <class>TableCtrl</class>
   <extends>QTableView</extends>
   <header>tablectrl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>firstBtnClicked()</slot>
 </slots>
</ui>
